package.json

```diff
@@ -106,9 +106,9 @@
     "swagger-ui": "2.1.4",
     "tv4": "^1.2.7",
     "velocity-animate": "^1.2.3",
-    "vue": "1.0.24",
+    "vue": "1.0.26",
     "vue-router": "0.7.13",
     "vue-strap": "^1.0.11",
-    "whatwg-fetch": "0.11.0"
+    "whatwg-fetch": "1.0.0"
```

Comment:
I saw that. But I spent the morning checking each dependency one by one (changelog and test).
By the way, it seems to have support for `URLSearchParams` in 1.0. I don't know if it's a polyfill or simply acceptance